plugin/src/main/java/org/autorefactor/refactoring/Refactorings.java

```diff
@@ -413,4 +413,13 @@ public BadLocationException call() throws Exception {
             throw ex;
         }
     }
+
+    /**
+     * Gets the ASTRewrite rewrite object.
+     *
+     * @return the ASTRewrite rewrite
+     */
+    public ASTRewrite getRewrite() {

```

Comment:
I needed this to create newlines inside ```ASTBuilder``` (```Statement newlinePlaceholder()```). I couldn't find an approach that did not use it ```rewrite```.